src/ReverseProxy/Configuration/CertificateConfigLoader.cs

```diff
@@ -23,13 +23,25 @@ public CertificateConfigLoader(IWebHostEnvironment hostEnvironment)
         }
 
         /// <inheritdoc/>
-        public X509Certificate2 LoadCertificate(CertificateConfigData certificateConfig)
+        public X509Certificate2 LoadCertificate(IConfigurationSection certificateSection)
         {
-            if (certificateConfig is null)
+            if (certificateSection?.Exists() is false)
```

Comment:
This PR uses `is false` in a bunch of places when it really wants to check if the `bool?` `!= true`.

```suggestion
            if (certificateSection?.Exists() != true)
```

Reply:
I'm just going to remove all of these nullable checks. The section is never null anyways.